### DISCUSSION TITLE
Problem: GRLB program working, lasser no working

### DISCUSSION THREAD

**Discussion by Valerijus67** [arkypita/LaserGRBL#1094] [Q&A]
> 
![prblem Ortur](https://user-images.githubusercontent.com/69475435/89931370-bdcce580-dc14-11ea-8815-a806f087f12c.jpg)

**Comment by StuartB4** [comment]
I notice on the 5th line down in the image it says "CLR:C:\Windows\Microsoft.NET\Framework\v2.0.50727.


You might need to update .NET Framework to a higher version. I think it needs v3.5 or later.
You can do that easily here:
https://dotnet.microsoft.com/download/dotnet-framework/net48

**Comment by Valerijus67** [comment]
Hi, you helped me. No more problems. Thanks

2020-08-11, an, 21:18 Stuart <notifications@github.com> rašė:

> I notice on the 5th line down in the image it says
> "CLR:C\Windows\Microsoft.Net\Framework\v2.0.50727.
>
> You might need to update .NET Framework to a higher version. You can do
> that easily here:
> https://dotnet.microsoft.com/download/dotnet-framework/net48
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/arkypita/LaserGRBL/issues/1094#issuecomment-672156273>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQSBY25GIF3FSVB3CX2BDXLSAGDPBANCNFSM4P3KCAHQ>
> .
>


-- 
 Pagarbiai Valerijus

**Comment by StuartB4** [comment]
Ok, that's good. Glad you got it sorted out. Thanks for letting me know. :+1:
